File: 2019-10-07/p2-fibonacci.py
from jutge import read

def fibonacci(n):
    """retorna l'n-èsim nombre de fibonacci, suposant n>=0 i n enter."""
    if n <= 1:
        return n
    else:
        return fibonacci(n - 1) + fibonacci(n - 2)

def main():
    n = read(int)
    print(fibonacci(n))

main()

File: 2019-10-07/p2-fibonacci.cc
#include <iostream>
using namespace std;

// retorna l'n-èsim nombre de fibonacci, suposant n>=0.
int fibonacci(int n) {
    if (n <= 1) return n;
    else return fibonacci(n - 1) + fibonacci(n - 2);
}

int main() {
    int n;
    cin >> n;
    cout << fibonacci(n) << endl;
}
